src/__tests__/main.test.ts

```diff
@@ -69,6 +69,26 @@ it('considers allow-one-liners.', () => {
   expect(mockSetFailed.mock.calls).toEqual([]);
 });
 
+it('considers skip-body-check.', () => {
+  (commitMessages.retrieve as any).mockImplementation(() => [
+    'Change SomeClass to OtherClass\n' +
+      '\n' +
+      'Change SomeClass to OtherClass.' +
+      'This replaces the SomeClass with OtherClass in all of the module ' +
+      'since Some class was deprecated.'
+  ]);
+
+  const mockSetFailed = jest.fn();
+  (core as any).setFailed = mockSetFailed;
+
+  (core as any).getInput = (name: string) =>
```

Comment:
(Please see the unit test that fails for the context.) This seems to be set for all the subsequent unit tests and triggers the unit tests below to fail. 

I'm not that experienced with Jest -- do you know what would be an idiomatic way to mock `core` such that individual unit tests can be run in parallel and don't affect each other?

A quick search lead me to this StackOverflow question: https://stackoverflow.com/questions/67260803/is-it-possibly-for-jest-mocks-to-be-overwritten-by-each-other

It seems to me that the right approach would be to use `mockImplementationOnce`. Please give me your feedback first and then I'll re-write the `main.test.ts` first before we make changes in this pull request.

Reply:
Actually, on a second read, I should prefix the test with `beforeEach` and reset the mocks. I'll try to do that today and come back to you once I fixed the tests.